Source/ASNetworkImageNode.mm

```diff
@@ -888,7 +888,7 @@ - (void)_lazilyLoadImageIfNecessary
               [delegate imageNodeDidLoadImageFromCache:self];
             }
             as_log_verbose(ASImageLoadingLog(), "Decached image for %@ img: %@ url: %@", self, [imageContainer asdk_image], URL);
```

Comment:
Would it be helpful to include the "is from memory" bit in this log message?

Reply:
@jparise Done thanks.